x-pack/plugins/observability_onboarding/public/components/app/custom_logs/wizard/install_elastic_agent.tsx

```diff
@@ -64,10 +61,7 @@ export function InstallElasticAgent() {
 
   const { data: monitoringRole, status: monitoringRoleStatus } = useFetcher(
     (callApi) => {
-      if (
-        CurrentStep === InstallElasticAgent &&
```

Comment:
so glad your change let's us get rid of this check to avoid the double API calls.